ui/components-react/package.json

```diff
@@ -147,10 +147,10 @@
       "ts-node/register"
     ],
     "check-coverage": true,
-    "statements": 99.9,
-    "branches": 99.9,
+    "statements": 100,
```

Comment:
Why change the threshold if only to add istanbul ignore commands ?

Reply:
Occasionally when we run rush update we pull in new packages that cause 100% test coverage to fail despite no changes to our own code. See #4406 for latest example.

Reply:
I believe this was caused by the PRs mentioned in the description of this PR and not because of the new package.